bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -3668,42 +3677,48 @@ public long getBlackboardArtifactsTypeCount(int artifactTypeID, long dataSourceI
 	 *                          within tsk core and artifacts could not be
 	 *                          queried
 	 */
+	@Deprecated
```

Comment:
We should at a @ deprecated item to the doc as well, and it should refer developers to the method that they should use instead. This comment applies to all of the methods given @ Deprecated annotations by this PR.